docs/Processes.md

```diff
@@ -221,8 +229,63 @@ the gateway api. Use .api.p“.gw.\*” for more details.
 | .gw.asyncexecjpt\[query; servertypes; joinfunction; postback; timeout\] | Execute the specified query against the required list of servers. Use the specified join function to aggregate the results. If the postback function is not set, the client must block and wait for the results. If it is set, the result will be wrapped in the specified postback function and returned asynchronously to the client. The query will be timed out if the timeout value is exceeded. |
 
 
-For the purposes of demonstration, assume that the queries must be run
-across an RDB and HDB process, and the gateway has one RDB and two HDB
+### Client Call Examples
+
+Here are a couple of examples for using client calls via a handle to the gateway process. 
+It is important to note if asynchronous calls are to be made, the .gw.synccallsallowed 
+variable needs to be set to 0b before running any async calls.
+To do this use
+`q) h(".gw.synccallsallowed:0b")`.
+
+#### Calls to the RDB only
+For synchronous calls
+```
+// To return the avg price per sym for the day so far
+q) h(`.gw.syncexec;"select avp:avg price by sym from trade where time.date=.z.d";`rdb)
+
+// Using the hloc function
+q) h(`.gw.syncexec;"hloc[2020.01.08;2020.01.08;10]";`rdb)
+
+// Returns following table
+sym  time                         | high   low    open   close  totalsize vwap
+----------------------------------| ----------------------------------------------
+AAPL 2020.01.08D00:00:00.836801000| 103.62 103.62 103.62 103.62 88        103.62
+AAPL 2020.01.08D00:00:01.804684000| 103.64 103.64 103.64 103.64 86        103.64
+AAPL 2020.01.08D00:00:02.405682000| 103.86 103.86 103.86 103.86 90        103.86
+AAPL 2020.01.08D00:00:03.005465000| 104.06 104.06 104.06 104.06 78        104.06
+AAPL 2020.01.08D00:00:03.404383000| 103.9  103.9  103.9  103.9  49        103.9
+..
+```
+For asynchronous calls
+```
+// To return the sum size per sym for the day so far
+q) neg[h](`.gw.asyncexec;"select sum size by sym from trade";`rdb);h[]
+```
+
+#### Calls to the HDB only
+For synchronous calls
+```
+// For the high, low, open and close prices of the day before
+q) h(`.gw.syncexec;"select h:max price, l:min price, o:first price, c:last price by sym from trade where date=.z.d-1";`hdb)
+```
+For asynchronous calls
+```
+q) neg[h](`.gw.asyncexec;"`$last .z.x";`hdb);h[]
+```
+
+#### Calls to the HDB and RDB
+For synchronous calls
+```
+q) h(`.gw.syncexec;"$[{`date in cols x}[trade]; select from trade where date within (.z.d-2;.z.d-1); select from trade]";`rdb`hdb)
```

Comment:
- Should we use `.proc.proctype` to mimic other examples on the doc?

- If hardcoding the table here I don't think you need the lamba for the conditional.